src/aselo-webchat.tsx

```diff
@@ -16,10 +15,23 @@ import { applyMobileOptimization } from './mobile-optimization';
 import { aseloReducer } from './aselo-webchat-state';
 import { subscribeToChannel } from './task';
 import { addContactIdentifierToContext } from './contact-identifier';
+import type { Configuration } from '../types';
 
 updateZIndex();
 
+export const getCurrentConfig = (): Configuration => {
+  try {
+    // eslint-disable-next-line no-unsanitized/method, global-require
+    const { config } = require(`./config`);
+    return config;
+  } catch (err) {
+    console.error(`Failed trying to load config file ${webpack.env.CONFIG}`, err);
+    throw err;
+  }
+};
```

Comment:
We could statically `import` this file, so TS will prevent compiling if it's missing (instead than runtime error), but in that case build step must be performed before running linter/tests, or it will fail pointing the invalid reference.
Should we go that way?

Reply:
I think this is fine if the tradeoff is having to run build before lint/test.

Reply:
I think this is fine if the tradeoff is having to run build before lint/test.